PixiEditor/Models/Undo/StorageBasedChange.cs

```diff
@@ -14,45 +16,64 @@ namespace PixiEditor.Models.Undo
     /// </summary>
     public class StorageBasedChange : IDisposable
     {
-        public static string DefaultUndoChangeLocation => Path.Join(Path.GetTempPath(), "PixiEditor", "UndoStack");
+        public static string DefaultUndoChangeLocation { get; } = Path.Join(Path.GetTempPath(), "PixiEditor", Guid.NewGuid().ToString(), "UndoStack");
```

Comment:
Maybe there should be a subfolder for the sessions, something like `Temp/PixiEditor/Sessions/GUID/UndoStack`

Reply:
Actually the only purpose of the PixiEditor folder is for us to have an easy way to find it during debugging

So I don't see why we would need another folder

Reply:
Well, it would make it easier to debug, also I think that the session folder should be deleted when PixiEditor closes